components/Svg.tsx

```diff
@@ -0,0 +1,13 @@
+interface SvgProps {
+    src : string,
+    className: string
+}
+const Svg = ({src, className} : SvgProps) => {
```

Comment:
this is not an SVG component, image is used for this, if you are not going to use an SVG,
you can directly call an image element in the component that use this